packages/grid/_modules_/grid/components/containers/GridRootStyles.ts

```diff
@@ -186,6 +186,7 @@ export const GridRootStyles = styled('div', {
     [`& .${gridClasses.row}`]: {
       display: 'flex',
       width: 'fit-content',
+      breakInside: 'avoid',
```

Comment:
Should we leave a comment on why?

```suggestion
      breakInside: 'avoid', // Avoid the row to be broken in two different print pages.
```

Reply:
If the CSS property seems to obscure to be self explanatory then we can add a comment yes :+1: